hs_core/management/commands/fix_resourcefile_duplicates.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+
+"""
+Fix resource file duplicates
+
+This checks that there is only one ResourceFile for each iRods file
+
+* By default, prints errors on stdout.
+* Optional argument --log instead logs output to system log.
+"""
+
+from django.core.management.base import BaseCommand
+from django.db.models import Count
+from hs_core.models import ResourceFile
+
+
+class Command(BaseCommand):
+    help = "Fix ResourceFile duplicates."
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--dryrun',
+            action='store_true',
+            dest='dryrun',
+            default=False,
+            help='Find duplicates but dont fix them',
+        )
+
+    def handle(self, *args, **options):
+        dry_run = options['dryrun']
+
+        dup_resource_files = ResourceFile.objects.values('resource_file', 'object_id') \
+            .annotate(count=Count('id')) \
+            .values('resource_file', 'object_id') \
```

Comment:
line 32 already has it.

Reply:
thanks, when I didn't have this "extra" `.values('resource_file', 'object_id')` I was getting errors for the subquery. I will take another look to understand why...

Reply:
Worked this time. Removed it.